README.md

```diff
@@ -0,0 +1,23 @@
+# socketburner
+
+A WebSocket script mirroring service for the game [Bitburner](https://github.com/danielyxie/bitburner).
+
+---
+
+### Dependencies
+* [`nodejs`](https://nodejs.org/)
+* [`yarn`](https://yarnpkg.com/)
+
+### Installation and Usage
+#### Linux
+```
+git clone https://github.com/MasonD/socketburner.git
+cd socketburner/
+yarn start
```

Comment:
yarn start won't work without first running `yarn` which defaults to `yarn install` and downloads dependencies. Might want to include `yarn install` somewhere in the instructions

Reply:
Ahh right. Is the latest commit okay now?